helm/charts/efs-provisioner/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
Is there any license related injecting their exact helm files ? Is this the official helm chart for efs

Reply:
Yes, this is the official EFS chart